partners_api/partners_api_tests/uber_tests.cpp

```diff
@@ -0,0 +1,131 @@
+#include "testing/testing.hpp"
+
+#include "geometry/latlon.hpp"
```

Comment:
Move this below partners_api. Usually we follow inverse-dependency order for inclusions, as for libraries in *.pro files. I.e. the most dependent folders first (like generator, search etc), the most basic last (like std or base).